pykubectl/objects.py

```diff
@@ -133,9 +133,10 @@ def execute(self, attempts=30):
             phase = self.get()['status']['phase']
             if phase == 'Failed':
                 self._abort()
-                raise KubernetesException('{} execution failed'.format(self))
+                raise KubernetesException(f'{self} execution failed')
             if phase == 'Succeeded':
                 logging.info('successfully completed')
+                logging.debug(self.logs(ignore_errors=True))
```

Comment:
Added this at the debug level

Reply:
we could put this under a argument flag instead of using debug? becuase this would still make a request even if you are not interested. not a biggie

Reply:
Hmm, I figured it was OK since performance isn't a strict concern in this workflow. Might simplify the workflow a bit to emit by default

Reply:
you don't know, depending on the use cases the logs might be gigantic. /shrug up to you

Reply:
Actually, I agree a flag makes sense, let me add that. It's similar to how a lot of CLI tools have a `--verbose` option I think...